src/scheduler.js

```diff
@@ -112,6 +113,18 @@ class Scheduler extends events.EventEmitter {
         taskId: hook.nextTaskId,
         time: new Date(),
       };
+
+      try {
+        await this.LastFire.create({
```

Comment:
Note that `triggerHoookCommon` in `api.js` also modifies the lastFire property of the Hook entity.  It might make sense to abstract that out to a common function that can do both `hook.modify` and `lastFire.create`.  In fact, I'm not sure why the TaskCreator doesn't do it?

Reply:
yes i think it's better then to append lastfire info in `taskcreator`

Reply:
or pass a callback function to `taskcreator.fire`.